test/test_gpu_transpose.cpp

```diff
@@ -39,7 +40,20 @@ TensorView* transposeMaybeInplace(
 
 } // namespace
 
-using TransposeTest = NVFuserTest;
+class TransposeTest : public NVFuserTest {
+  void SetUp() override {
+    NVFuserTest::SetUp();
+    // For convenience, disable MarkAliasPass. Many tests in this file run a
+    // fusion that consists of `transpose` only. MarkAliasPass would turn those
+    // fusions into a no-op, skipping the transpose scheduler.
+    optimization::MarkAliasPass::setEnabled(false);
+  }
+
+  void TearDown() override {
+    optimization::MarkAliasPass::setEnabled(true);
```

Comment:
nitpick: we probably want to restore the old flag rather than blindly setting it to true.

Reply:
Good catch! Done.